amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2495,7 +2497,201 @@ paths:
+        '404':
+          description: recordCategoryId does not exist
+        '409':
+          description: Retention schedule already exist for the given recordCategoryId
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    get:
+      tags:
+        - retention-schedule
+      summary: Get the retention schedule for a record category
+      description: |
+        Get the retention schedule for a record category.
+        
+        You can use the **include** parameter (include=actions) to return additional information.
 
+      operationId: getRetentionScheduleList
+      parameters:
+        - $ref: '#/parameters/recordCategoryIdParam'
+        - $ref: '#/parameters/retentionScheduleIncludeParam'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/RetentionScheduleResponseList'
+        '400':
+          description: |
+            Invalid parameter: recordCategoryId is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to get retention schedule
+        '404':
+          description: recordCategoryId does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/retention-schedule/{retentionScheduleId}/retention-steps':
+    post:
+      tags:
+        - retention-schedule
+      summary: Create a step in the retention schedule
+      description: |
+        Create a step in the retention schedule.
+        The valid steps types are:
+          * **retain**
+          * **cutoff**
+          * **accession**
+          * **transfer**
+          * **destroy**
+        
+        Order of steps:
+          * "retain" or "cutoff" should be first
+          * can't use "cutoff" after "transfer" or "accession"
+          * "transfer" action is allowed multiple times
+          * "destroy" must be the last action
+        
+        The **combineDispositionStepConditions** property is only valid for **accession** step.
+        This is used to specify whether to combine the period condition and events for the step execution or only consider one of them.
+        For example:
+          **periodCondition**: After a period of 2 months
+          **eventsCondition**: Case Closed event
+          This flag can be used to consider only (**periodCondition** or **eventsCondition**) or both of them at once.
+        
+        For example, the following JSON body to create a step in the retention schedule:
+        ```JSON
+          {
+              "name":"retain",
+              "periodAmount": 2,
+              "period":"month",
+              "periodProperty":"cm:created",
+              "combineDispositionStepConditions": false,
```

Comment:
Above we said this was only valid for "accession" steps. Either change the step type to "accession" or remove this property.

Reply:
changed the step type to accession.